docs/user-guide/fhir-resource-tooling.md

```diff
@@ -42,3 +42,18 @@ $> java -cp fhir-resource-tooling-0.0.1-SNAPSHOT-shaded.jar com.ibm.cohort.tooli
 ```
 
 Please note, if you intend to use custom mappings those mappings must be provided to load the value set with this tool.
+
+There is also the ability to write the json representation of the value set either locally or to cos, in the case that you would like to check that it looks how you would expect. In this case, rather than the server configuration, you would need to specify either COS, LOCAL, or BOTH with the --output-locations flag.
+
+
+#####Parameters
+
+* -m/--measure-server: The path to the JSON configuration data for the FHIR server connection that will be used to retrieve measure and library resources. If this is provided, `output-locations` must be **NONE**.
```

Comment:
Pretty much everywhere else in the user guide, if we are displaying the program usage, we use a triple-backtick pre-formatted box and just copy/paste in the program usage. Not sure this is a hard request to change since you are using some bold formatting, but inconsistency bothers me.